doc/man3/SSL_CONF_cmd.pod

```diff
@@ -415,12 +417,13 @@ B<signature_scheme>. For the default providers shipped with OpenSSL,
 B<algorithm> is one of B<RSA>, B<DSA> or B<ECDSA> and B<hash> is a supported
 algorithm OID short name such as B<SHA1>, B<SHA224>, B<SHA256>, B<SHA384>
 or B<SHA512>.
-Note: algorithm and hash names are case sensitive.
+Note: the algorithm names (but not the hash names) are case-insensitive.
```

Comment:
Nit: I'm not sure I understand what the word "algorithm" refers to now: Both signature algorithms as well as hash algorithms have names in my understanding. So would this formulation be more clear (but I wonder whether it's really intended that way looking at the code below?):

> Note: Signature algorithm names are case-insensitive. Hash algorithm names are case sensitive.

? And maybe an example (if I understand it right)? "For example, `ecdsa_secp256r1_sha256` and `ECDSA_secp256r1_sha256` denote the same signature scheme while `ecdsa_secp256r1_sha256` and `ecdsa_secp256r1_SHA256` denote different signature schemes.

OR: Do do you intend to state 

"Note: `algorithm` and `signature_scheme` names are case-insensitive; `hash` names are case sensitive."

?

Reply:
Hash names are case-sensitive **only** in the `KEYALG+HASHALG` construct (legacy two-part TLS signature algorithm).
When part of a monolithic *signature scheme*, the entire name is case-insensitive, even if part of that name is a digest algorithm name.

I guess this wasn't clear enough...

Reply:
Hence my second suggestion to re-phrase. It now is clear to me -- others might read this in the future and could likewise wonder.